Parscore.js

```diff
@@ -0,0 +1,75 @@
+function handleFileSelect(evt) {
+    evt.stopPropagation();
+    evt.preventDefault();
+    var files = evt.dataTransfer.files; // FileList object.
+    var file = files[0];
+    //checks to see if they dropped a valid filetype
+    if((file.type != "text/csv") && (file.type != "text/plain") && (file.type != "application/vnd.ms-excel")){
+        alert("Please only drag and drop CSV files from Blackboard or TXT files from Parscore." + file.type);
+        return;
+    }
+    document.getElementById('list').innerHTML = '<p>' + file.name + '<p>';
```

Comment:
You are using jQuery in this project, so some of these statements, like this one, could be optimized.